src/styles/index.scss

```
@@ -1,3 +1,78 @@
-body {
+body,
+html {
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   margin: 0;
+  background-color: #f0f0f0;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
+
+  &__center {
+    position: absolute;
+    width: 5vmin;
+    height: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 10;
+  }
+
+  &__hand {
+    position: absolute;
+    bottom: 50%;
+    top: 50%;
+    left: 50%;
+    transform-origin: bottom;
+    transform: translateX(-50%) translateY(-100%);
+  }
+
+  &__hand--minutes {
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    animation: minutes 3600s steps(60) infinite;
+  }
+
+  &__hand--seconds {
+    width: 1.5vmin;
+    height: 38vmin;
+    background-color: #2c8000;
+    animation: seconds 60s linear infinite;
+  }
+}
+
+.stopwatch--speed-up .stopwatch__hand--minutes {
+  animation: minutes 600s steps(60) infinite;
+}
+
+.stopwatch--speed-up .stopwatch__hand--seconds {
+  animation: seconds 10s linear infinite;
+}
+
+@keyframes minutes {
+  from {
+    transform: translateX(-50%) translateY(-100%) rotate(0deg);
+  }
+  to {
+    transform: translateX(-50%) translateY(-100%) rotate(360deg);
+  }
+}
+
+@keyframes seconds {
+  from {
+    transform: translateX(-50%) translateY(-100%) rotate(0deg);
+  }
+  to {
+    transform: translateX(-50%) translateY(-100%) rotate(360deg);
+  }
```

you can reuse single keyframe as they are identical
```suggestion
@keyframes rotate {
  from {
    transform: translateX(-50%) translateY(-100%) rotate(0deg);
  }
  to {
    transform: translateX(-50%) translateY(-100%) rotate(360deg);
  }
```